deploy/crds/pulpproject_v1beta1_pulp_crd.yaml

```diff
@@ -59,8 +86,8 @@ spec:
                     type: string
                     default: ReadWriteMany
                     enum:
-                      - ReadWriteMany
-                      - ReadWriteOnce
+                    - ReadWriteMany
+                    - ReadWriteOnce
```

Comment:
I actually prefer your indentation style, but [the default yamllint rules (spaces, indent-sequences) are for the the 2 spaces in the beginning.](https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.indentation) So that is what we have been using.